packages/vitest/src/node/cli-api.ts

```diff
@@ -44,6 +44,12 @@ export async function startVitest(
   if (typeof options.coverage === 'boolean')
     options.coverage = { enabled: options.coverage }
 
+  // running "vitest --coverage.all" or "vitest --coverage.100"
+  // @ts-expect-error Element implicitly has an any type for options.coverage["100"]
+  if (typeof options.coverage === 'object' && !('enabled' in options.coverage) && (options.coverage['100'] === true
+    || Object.keys(options.coverage).length > 1))
+    options.coverage.enabled = true
+
```

Comment:
So this sets `coverage.enabled` when any coverage options are passed as CLI arguments? We don't do that for any other options either so it should not be done here either.

Reply:
It's done on the browser check [here](https://github.com/vitest-dev/vitest/blob/aa6b6f501889f0255ed9b00375605c257b00b830/packages/vitest/src/node/cli-api.ts#L62) that's why I added it =P
But I'll update it. 
Should I update the docs to let the user know that he needs to use the --coverage.enabled when using other coverage options ?

Reply:
Let's leave this change out of this PR and include only the fix for the crash. 
This change might be good to do in another PR though.